packages/keyring-controller/src/KeyringController.ts

```diff
@@ -429,6 +431,9 @@ export class KeyringController extends BaseController<
    * @returns Promise resolving to a signed message string.
    */
   signMessage(messageParams: PersonalMessageParams) {
+    if (!messageParams.data) {
+      throw new Error("Can't sign an empty message");
```

Comment:
I thought this error could be pulled upstream. A different error is now being thrown by `signMessage` than was previously documented in the tests related to a `BigInt` failure and I figured it was better to provide a more accurate error to indicate that a falsy data value here is expected to fail.

Reply:
Yeah this seems like a good idea to me. Both the old and new error messages from `eth-keyring-controller` seem unhelpful.